src/_components/User/AddSecurityUserContainer.jsx

```diff
@@ -0,0 +1,113 @@
+import * as React from 'react';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+import AddUserComponent from '../../components/User/AddSecurityUserComponent';
+import { ToastMessage } from '../../_utils/ToastMessage';
+import { addSecurityUser } from '../../_actions/users.actions';
+import resetToastMessageContent from '../../_actions/toastMessage.actions';
+
+class AddSecurityUserContainer extends React.Component {
+  state = {
+    firstName: '',
+    lastName: '',
+    phoneNumber: '',
+    badgeNumber: '',
+    email: '',
+    saveButtonState: false
+  };
+
+  static getDerivedStateFromProps(nextProps) {
+    const { toastMessageContent } = nextProps;
+    if (toastMessageContent.type) {
+      if (toastMessageContent.type === 'success') {
+        ToastMessage.success({
+          message: toastMessageContent.message
+        });
+      } else if (toastMessageContent.type === 'error') {
+        ToastMessage.error({
+          message: toastMessageContent.message
+        });
+      }
+      nextProps.resetToastMessageContent();
+      nextProps.toggleModal();
+      return {
+        firstName: '',
+        lastName: '',
+        phoneNumber: '',
+        badgeNumber: '',
+        email: ''
+      };
+    }
+    return null;
+  }
+
+  onFirstNameChange = (event) => {
```

Comment:
Please consolidate `onFirstNameChange`, `onLastNameChange`, `onPhoneNumberChange`, `onEmailChange` and `onBadgeNumberChange` to use one onChange handler as described in [handling multiple inputs](https://reactjs.org/docs/forms.html#handling-multiple-inputs). 

This means that all the inputs handled by the mention onChange handlers will have a `name` attribute, which will be the same as the state variable. Then you will have only one onChange handler. E.g.
```
handleInputChange = (event) =>  {
  const { value, name } = event.target;
  this.setState({
    [name]: value
  });
}